.github/workflows/pr-check.yaml

```
@@ -84,3 +84,84 @@ jobs:
 
       - name: Run build
         run: yarn build
+  
+  e2e-tests:
+    name: bootc e2e tests
```

```suggestion
    name: e2e tests
```
we're inside the bootc repository anyway